docs/guides/platforms/windows/host-avalonia-controls-in-winforms.md

```diff
@@ -0,0 +1,56 @@
+---
+id: host-avalonia-controls-in-winforms
+title: Host Avalonia controls in Windows Forms
+---
+
+# Host Avalonia controls in Windows Forms
+
+Avalonia controls can be hosted in Windows Forms applications. This enables a step-by-step migration of existing Windows Forms applications to Avalonia.
+
+## Overview
+
+An exemplary Windows Forms application with Avalonia controls requires at least three projects:
+
+1. `YourApp` Cross platform library where you put your Avalonia controls 
+2. `YourApp.Desktop` Executable Avalonia application, required for the Visual Studio Avalonia designer
+3. `YourApp.WinForms` Your existing Windows Forms application
+
+## Prerequisites
+
+- Visual Studio 2022
+- Avalonia extension for Visual Studio 2022
```

Comment:
Pretty sure these two are optional. You also can use Rider for WinForms. Or even VSCode.
Windows OS itself is only what's necessary (at least to run the project).